src/Sulu/Bundle/AdminBundle/Resources/js/views/List/toolbarActions/ExportToolbarAction.js

```diff
@@ -0,0 +1,125 @@
+                                    <SingleSelect.Option value=" ">&nbsp;</SingleSelect.Option>
+                                </SingleSelect>
+                            </Form.Field>
+                        </Form.Section>
+                        <Form.Section colSpan={6}>
+                            <Form.Field
+                                description={translate('sulu_admin.escape_description')}
+                                label={translate('sulu_admin.escape')}
+                            >
+                                <SingleSelect onChange={this.handleEscapeChanged} value={this.escape}>
+                                    <SingleSelect.Option value={'\\'}>\</SingleSelect.Option>
+                                    <SingleSelect.Option value={'"'}>&quot;</SingleSelect.Option>
+                                </SingleSelect>
+                            </Form.Field>
+                            <Form.Field
+                                description={translate('sulu_admin.new_line_description')}
+                                label={translate('sulu_admin.new_line')}
+                            >
+                                <SingleSelect onChange={this.handleNewLineChanged} value={this.newLine}>
+                                    <SingleSelect.Option value={'\\n'}>\n</SingleSelect.Option>
+                                    <SingleSelect.Option value={'\\r\\n'}>\r\n</SingleSelect.Option>
+                                    <SingleSelect.Option value={'\\r'}>\r</SingleSelect.Option>
+                                </SingleSelect>
+                            </Form.Field>
+                        </Form.Section>
+                    </Form>
+                </div>
+
+            </Overlay>
+        );
+    }
+
+    getToolbarItemConfig() {
+        return {
+            icon: 'su-download',
+            label: translate('sulu_admin.export'),
+            onClick: action(() => {
+                this.showOverlay = true;
+            }),
+            type: 'button',
+        };
+    }
+
+    @action handleClose = () => {
+        this.showOverlay = false;
+    };
+
+    @action handleDelimiterChanged = (value: string) => {
+        this.delimiter = value;
+    };
+    @action handleEnclosureChanged = (value: string) => {
+        this.enclosure = value;
+    };
+    @action handleEscapeChanged = (value: string) => {
+        this.escape = value;
+    };
+    @action handleNewLineChanged = (value: string) => {
+        this.newLine = value;
+    };
+
+    @action handleConfirm = () => {
+        window.location.assign(resourceEndpointRegistry.getListUrl(this.listStore.resourceKey) + '.csv' +
+            buildQueryString(
+                {
+                    flat: true,
+                    delimiter: this.delimiter,
+                    escape: this.escape,
+                    enclosure: this.enclosure,
+                    newLine: this.newLine,
+                }
```

Comment:
All of this probably can be passed to the second argument of `getListUrl` now, since it has been refactored.